docs/campaign/quests/substrate-pallets.md

```diff
@@ -17,32 +17,159 @@ If you are unsure which chain state function to call, you can also use this step
-    ![Substrate Pallets Quest on AirLyft](../../images/pallets-3.png)
 2. Fill in the basic quest information (name, description, cFuel, and XP).
-3. To configure the task add the following details:
-    - Select **Subsocial** from the Blockchain dropdown.
-    - Select **Space Follows(spaceFollows)** from the Storage entry to query dropdown. Then choose **spaceFollowedByAccount(spaceFollowedByAccount)**.
-4. The, toggle the button next to Account32 input box, and select **User's connected wallet** in the address field.
-5. Enter the ID of the space you want to check, in our case "1".
-    ![Pallet Task Configuration](../../images/pallets-4.png)
-6. In the output condition, write a JavaScript condition for task validation. You can customize it based on your needs, but in our case we'll leave it as it is.
-7. Then click on **Add Task** button. That's it your Substrate Pallets quest has been created.
-    ![Pallet Task Output condition](../../images/pallets-5.png)
+   ![](../../images/palletAssetHubNft.png)
+
+3. To configure the **Task Condition & Recurrence**, refer to the [Task Condition & Recurrence](../task-condition-and-recurrence.md) page.
+
+4. Select your **Blockhain** and then enter the **NFT ID** and **Requirement Type**.
+
+5. Then click on **Add Task** button. That's it your Substrate Pallets quest has been created.
+
+#### Substrate Raw Querry
+
+- This option verifies whether a user has bonded their tokens to a staking pool, an essential process in decentralized finance and staking ecosystems. By using this task, you can encourage users to engage in staking pools, supporting network security while earning passive income.
+
+1. Fill in the basic quest information (name, description, cFuel, and XP).
+
+2. To configure the task add the following details:
+
+- Select **Subsocial** from the Blockchain dropdown.
+- Select **Space Follows(spaceFollows)** from the Storage entry to query dropdown. Then choose **spaceFollowedByAccount(spaceFollowedByAccount)**.
+
+3. The, toggle the button next to Account32 input box, and select **User's connected wallet** in the address field.
+
+4. Enter the ID of the space you want to check, in our case "1".
+   ![Pallet Task Configuration](../../images/pallets-4.png)
+
+5. In the output condition, write a JavaScript condition for task validation. You can customize it based on your needs, but in our case we'll leave it as it is.
+
+6. Then click on **Add Task** button. That's it your Substrate Pallets quest has been created.
+   ![Pallet Task Output condition](../../images/pallets-5.png)
+
+#### Substrate Bond to Pool
+
+- In Proof-of-Stake networks, users can delegate their tokens to validators who help secure the blockchain. This option checks if a user has nominated a validator, ensuring they are actively participating in governance and network security. Campaigns using this task can reward users who contribute to the decentralization and efficiency of the network.
```

Comment:
not needed `In Proof-of-Stake networks, users can delegate their tokens to validators who help secure the blockchain.`

Reply:
not needed `Campaigns using this task can reward users who contribute to the decentralization and efficiency of the network.`